Assets/Scripts/Entity/Player.cs

```diff
@@ -10,6 +10,9 @@ class Player : AbstractEntity
 
         public int collectible;
         public Text counter;
+        public float enemyBumpPowerY = 10f;
```

Comment:
Enemy bump power should come from the enemy and not defined in the player

Reply:
Moved bump power to enemy script.